### DISCUSSION TITLE
Diagnostics Include LSP Info Results

### DISCUSSION THREAD

**Discussion by ybbond** [zed-industries/zed#23974] [Feature Requests]
### Summary

currently we have the option to

```json
{
  // Diagnostics configuration.
  "diagnostics": {
    // Whether to show warnings or not by default.
    "include_warnings": true
  },
}
```

I'd like for Zed to also has option to show info

```json
{
  // Diagnostics configuration.
  "diagnostics": {
    // Whether to show infos or not by default.
    "include_infos": false
  },
}
```

Some people would like to jump to as verbose LSP result as possible

**Comment by probably-neb** [comment]
Hey! Thanks for bringing this up to us. We've decided to start moving feature/enhancement requests like this one to discussions. Feel free to keep discussing what this might look like and how it would help make Zed a better editor for you there.
